### ISSUE TITLE
multiple paths are not allowed

### ISSUE THREAD

**Issue by yegor256** [g4s8/xcop-action#3]
I would like to have an ability to do this:

```
- uses: g4s8/xcop-action@master
  with:
    license: MY_LICENSE.txt
    files: "src/*.xml"
```

and this:

```
- uses: g4s8/xcop-action@master
  with:
    license: MY_LICENSE.txt
    files: 
      - "src/*.xml"
      - "src/*.xsl"
```

At the moment, the second option is not possible.

**Comment by yegor256** [comment]
@g4s8 can you please help?

**Comment by yegor256** [comment]
@g4s8 need help?

**Comment by baudoliver7** [comment]
@g4s8 Hi, please assign

**Comment by g4s8** [comment]
@baudoliver7 feel free to submit a pull request. But keep in mind, that GitHub actions don't allow to pass complex structure to `with` parameters, only key value string pairs are allowed. Most probably you will need to provide multiple paths as coma separated string (or use other separator) and parse it from code or script.

**Comment by baudoliver7** [comment]
@g4s8 Well noticed

**Comment by g4s8** [comment]
This should be fixed by #7
